.github/workflows/alpha-nightly.yaml

```diff
@@ -1,8 +1,12 @@
 name: Alpha Nightly
+env:
+  # This must be defined for the bash redirection
+  GOOGLE_APPLICATION_CREDENTIALS: 'jade-dev-account.json'
+  # This must be defined for the bash redirection
+  GOOGLE_SA_CERT: 'jade-dev-account.pem'
```

Comment:
Do we need this on alpha-promotion.yaml as well?

Reply:
Luckily it doesn't use the `datarepo-actions` repository at all!

Reply:
ah ok